support/client/lib/vwf/view/threejs.js

```diff
@@ -172,9 +172,9 @@ define( [ "module",
                 setActiveCamera.call(this, this.state.scenes[ this.kernel.application() ].camera.ID);
             }
         
-            if(this.state.nodes[childID] && this.state.nodes[childID].threeObject instanceof THREE.Object3D) {
-                this.nodes[childID] = {id:childID,extends:childExtendsID};
-            }
+            // if ( this.state.nodes[childID] && this.state.nodes[childID].threeObject instanceof THREE.Object3D ) {
+            //     this.nodes[childID] = {id:childID,extends:childExtendsID};
+            // }
         },
```

Comment:
If this is no longer needed, I vote to go ahead and remove it rather than comment it out.